src/main/docker/bamboo/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM atlassian/bamboo-server:7.2.3
+FROM atlassian/bamboo-server:7.1.2
```

Comment:
Why did you changed this file in this PR

Reply:
Oh no I accidently commited that